rpcserver.go

```diff
@@ -168,9 +168,60 @@ var (
 			Action: "write",
 		},
 	}
+)
+
+// rpcServer is a gRPC, RPC front end to the lnd daemon.
+// TODO(roasbeef): pagination support for the list-style calls
+type rpcServer struct {
```

Comment:
Is moving this struct necessary? Diff would be much smaller if not.

Reply:
@halseth  I haven't moved the struct, it is the diff algo probably. Struggled with it and couldn't force it to be smaller than this.

Reply:
It seems like we can avoid the bigger diff by moving `mainRPCServerPermissions` above `type rpcServer struct`.

Reply:
Yeah, keeping the struct below `mainRPCServerPermissions` should fix it?

Reply:
@halseth indeed. fixed.